mp/src/game/shared/neo/neo_gamerules.cpp

```diff
@@ -52,6 +52,14 @@ ConVar neo_sv_build_integrity_check_allow_debug("neo_sv_build_integrity_check_al
 									"If enabled, when the server checks the client hashes, it'll also allow debug"
 									" builds which has a given special bit to bypass the check.",
 									true, 0.0f, true, 1.0f);
+
+ConVar neo_sv_mirror_teamdamage_multiplier("neo_sv_mirror_teamdamage_multiplier", "2", FCVAR_REPLICATED, "The damage multiplier given to the friendly-firing individual. Set value to 0 to disable mirror team damage.", true, 0.0f, true, 100.0f);
```

Comment:
Could we perhaps have this default to 0 for debug builds? Because typically when testing stuff, you don't want the game to auto-slay or kick you. Could be just a simple `_DEBUG` ifdef for the default values.